From 888c1e0ed317549c821babb10e72f731dbb32a73 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Wed, 13 Dec 2023 11:29:00 +0100
Subject: [PATCH 0180/1695] dts: stm32: enable stm32-rcc driver for
 stm32mp135f-dk

Add support of pinctrl and clock for RCC node

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9587a0e3a64e1161b6d0ea4724b0bb2665ab55fc dts: stm32: enable stm32-rcc driver for stm32mp135f-dk]
Change-Id: I8ca48e12bbbea67b2d31cc7cd3e7f7536f983714
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 920db48b2..48c29512f 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -429,6 +429,11 @@
 			};
 		};
 	};
+
+	st,rcc {
+		compatible = "st,rcc";
+		pinctrl-0 = <&rcc_mco_pins_a>;
+	};
 };
 
 &rng {
-- 
2.47.2

